packages/core/content-releases/admin/src/components/ReleaseModal.tsx

```diff
@@ -255,12 +256,12 @@ const getTimezones = (selectedDate: Date) => {
     // a four digit string e.g. +05:00 or -08:00
     const utcOffset = getTimezoneOffset(timezone, selectedDate);
 
-    // Offset and timezone are concatenated with '_', so to split and save the required timezone in DB
-    return { offset: utcOffset, value: `${utcOffset}_${timezone}` } satisfies ITimezoneOption;
+    // Offset and timezone are concatenated with '&', so to split and save the required timezone in DB
+    return { offset: utcOffset, value: `${utcOffset}&${timezone}` } satisfies ITimezoneOption;
   });
 
   const systemTimezone = timezoneList.find(
-    (timezone) => timezone.value.split('_')[1] === Intl.DateTimeFormat().resolvedOptions().timeZone
+    (timezone) => timezone.value.split('&')[1] === Intl.DateTimeFormat().resolvedOptions().timeZone
```

Comment:
I would have made `'&'` a constant to make it clear what that is 😄 @madhurisandbhor

Reply:
I knew you will have something to say thats why I requested your approval 😄 and I waited. 
I have added a comment above though why concatenating with '&'.

Reply:
hahaha sorry, busy day, next time ping me on slack lol i dont mind 😄